apps/meteor/app/livechat/server/lib/LivechatTyped.ts

```diff
@@ -1807,6 +1808,8 @@ class LivechatClass {
 				Subscriptions.updateDisplayNameByRoomId(rid, name),
 			]);
 
+			void notifyListenerOnRoomChanges(rid);
```

Comment:
I think we need to "notify" outside this `if` as well, since the room was also changed on line 1796 when it called `LivechatRooms.saveRoomById`